apis/slo/v1alpha1/nodemetric_types.go

```diff
@@ -23,8 +23,25 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+type AggregationType string
+
+const (
+	// max is not welcomed since it may import outliers
+	AVG    AggregationType = "avg"
+	P99    AggregationType = "p99"
+	P95    AggregationType = "p95"
+	P90    AggregationType = "p90"
+	MEDIAN AggregationType = "median"
```

Comment:
Why not use p50 to keep consistently

Reply:
Maybe in the percentile statistical method, the median is generally used to represent p50